spec/groomer_spec.rb

```diff
@@ -0,0 +1,56 @@
+require './lib/groomer'
+require './lib/customer'
+require './lib/pet'
+
+RSpec.describe Groomer do
+  before(:each) do
+    @groomer = Groomer.new("The Hair Ball")
+  end
+
+  it "exists" do
+    expect(@groomer).to be_instance_of(Groomer)
+  end
+
+  it "has a name" do
+    expect(@groomer.name).to eq("The Hair Ball")
+  end
+
+  it "starts with no customers" do
+    expect(@groomer.customers).to eq([])
+  end
+
+  it "can add customers" do
+    joel = Customer.new("Joel", 2)
+    billy = Customer.new("Billy", 3)
+    @groomer.add_customer(joel)
+    @groomer.add_customer(billy)
+
+    expect(@groomer.customers).to eq([joel, billy])
+  end
+
+  it "can count the number of pets of a certain type" do
+    joel = Customer.new("Joel", 2)
+    samson = Pet.new({name: "Samson", type: :dog})
+    lucy = Pet.new({name: "Lucy", type: :cat})
+    joel.adopt(samson)
```

Comment:
Again, do we want to put `Pets` and `Customers` into the before(:each)?

Reply:
I just updated this - check out the latest changes.